crypto/src/hash.rs

```diff
@@ -69,24 +73,23 @@ impl AsRef<[u8]> for Hash {
 }
 
 /// Represents hash of Iroha entities like `Block` or `Transaction`. Currently supports only blake2b-32.
-#[derive(Debug, Encode, Decode, Serialize, Deserialize, Deref, DerefMut, Display)]
+// Lint triggers when expanding #[codec(skip)]
+#[allow(clippy::default_trait_access)]
+#[derive(Deref, DerefMut, Display, Decode, Encode, Deserialize, Serialize)]
 #[display(fmt = "{}", _0)]
+#[serde(transparent)]
 pub struct HashOf<T>(
     #[deref]
     #[deref_mut]
     Hash,
-    PhantomData<T>,
+    #[codec(skip)] PhantomData<T>,
```

Comment:
`PhantomData` is skipped by default with scale codec, but it doesn't hurt to be explicit

Reply:
also, it removes this field from the generated schema